docs/admin.rst

```diff
@@ -413,23 +413,42 @@ for how to enable error logging for the *Source Interface*.
 Immediately Apply a SecureDrop Update
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-SecureDrop will update and reboot once per day. However, if after a SecureDrop
-update `is announced`_ you wish to fetch the update immediately, you can SSH
-into each server (via ``ssh app`` and ``ssh mon``) and run:
-
-.. code:: sh
-
-  sudo cron-apt -i -s
-
-Depending on the nature of the update (e.g., if the ``tor`` package is upgraded
-and you are using SSH-over-Tor), your SSH connection may be interrupted, and you
+SecureDrop will update and reboot once per day. However, once a SecureDrop
+update `is announced`_ , you can opt to fetch the update immediately. Depending
+on the nature of the update (e.g., if the ``tor`` package is upgraded and you are
```

Comment:
(Nit) for clarity's sake, I would move the "Depending on the nature of the update, ..." sentence to the very end of these instructions (current line 451), optionally in a ``note``.   But I don't think it's worth doing unless other people request other changes as well.